sdks/python/apache_beam/transforms/core.py

```diff
@@ -2394,8 +2399,8 @@ class GroupBy(PTransform):
   """
   def __init__(
       self,
-      *fields,  # type: typing.Union[str, callable]
-      **kwargs  # type: typing.Union[str, callable]
+      *fields,  # type: typing.Union[str, typing.Callable]
```

Comment:
Is there any advantage to (untyped) `typing.Callable`?

Reply:
mypy does not accept `callable`: `Function "builtins.callable" is not valid as a type`

Reply:
Oh, yes, of course. callable is a function returning whether its argument is callable.